astropy/io/ascii/tests/test_cds.py

```diff
@@ -450,28 +464,31 @@ def test_write_extra_skycoord_cols():
 --------------------------------------------------------------------------------
  Bytes Format Units  Label     Explanations
 --------------------------------------------------------------------------------
- 1- 7  A7     ---    name    Description of name     
- 9-10  I2     h      RAh     Right Ascension (hour)  
+ 1- 7  A7     ---    name    Description of name
+ 9-10  I2     h      RAh     Right Ascension (hour)
 12-13  I2     min    RAm     Right Ascension (minute)
 15-27  F13.10 s      RAs     Right Ascension (second)
-   29  A1     ---    DE-     Sign of Declination     
-30-31  I2     deg    DEd     Declination (degree)    
-33-34  I2     arcmin DEm     Declination (arcmin)    
-36-47  F12.9  arcsec DEs     Declination (arcsec)    
-49-62  A14    ---    coord2  Description of coord2   
+   29  A1     ---    DE-     Sign of Declination
+30-31  I2     deg    DEd     Declination (degree)
+33-34  I2     arcmin DEm     Declination (arcmin)
+36-47  F12.9  arcsec DEs     Declination (arcsec)
+49-62  A14    ---    coord2  Description of coord2
 --------------------------------------------------------------------------------
 Notes:
 --------------------------------------------------------------------------------
-HD4760   0 49 39.9000000000 +06 24 07.999200000 12.4163 6.407 
+HD4760   0 49 39.9000000000 +06 24 07.999200000 12.4163 6.407
 HD81809 22 02 15.4500000000 -61 39 34.599996000 330.564 -61.66
```

Comment:
Looks like something went wrong with this table. I think the spaces at the end of the line were removed incorrectly.

Reply:
I'll fix that. The tests should be written in a way to not depend on trailing spaces. I've done that in other contexts.